Code_Sample_README.md

```diff
@@ -0,0 +1,83 @@
+# blackbaud-crm-sky
+A repository containing the code needed to extend Blackbaud CRM's Fundraiser on the Go SKY-based mobile app.
+
+## Code Samples
+
+Follow these instructions to extend Blackbaud SKY-based mobile apps.
+
+### Adding logic to an existing form
+
+When completing a step/interaction, the out of the box version does not have any custom logic. If you require a category when completing a step, you can customize frogger to do the same. Here are the steps:
+
+1. Determine what is needed to complete the functionality. For the category, we need to know the completed status code (function in apiContactReportOptions) and the current status of the step/interaction (already defined in `contactreport.js`). Since these already exist, we can use them to build our custom function.
+2. Edit `custom.js` to add the new function to make `category` required. Note that `apiContactReportOptions` needs to be injected in the factory to support this function :
+    ```javascript
```

Comment:
Nice code snippet! Please format tabs and spacing as if it were in production. Looks like there are a lot of extra indents here.